File: OOP/class.cpp
#include<iostream>
#include<string>

using namespace std;

class Car{
    private:
        string make;
        string model;
        int noPassengers;

    public:
        Car(string make, string model, int noPassengers){
            this->make = make;
            this->model = model;
            this->noPassengers = noPassengers;
        }

        void start(){
            cout<<"Car Started!";
        }

        void print(){
            cout<<"Make: "<<this->make<<endl;
            cout<<"Model: "<<this->model<<endl;
            cout<<"No Passengers: "<<this->noPassengers<<endl;
        }
};

int main(){
    Car polo("Volkswagen", "Polo", 5);
    polo.print();
}

File: Javascript/ES6/class.js
// Program to demonstrate the working of class in ES6

class Car {
	constructor(make, model, passengers, color, speed){
		this.make = make;
		this.model = model;
		this.passengers = passengers;
		this.color = color;
		this.maxspeed = speed;
	}
}

console.log(typeof Car);

let polo = new Car('Volkswagen', 'Polo', 5, 'Green', 200);
console.log(typeof polo);
console.log(polo);
